stable/kommander/templates/grafana/dashboards/k8s-resources-clusters.yaml

```diff
@@ -2677,8 +2668,8 @@ data:
         ]
       },
       "timezone": "",
-      "title": "Kubernetes / Compute Resources / Clusters",
-      "uid": "efa86fd1d0c121a26444b636a3f509a8",
+      "title": "Kubernetes / Compute Resources / Cluster [Global]",
+      "uid": "f0U1EswWk",
```

Comment:
Minor nit, but I'll not block on it. This doesn't seem like it's a `uid`. Doesn't even seem to be a `uuid` any more. Should this key be more descriptive?

Reply:
It's a grafana key so we can't change it, but this was the value automatically generated by grafana when I created the dashboard in the UI so I wouldn't worry too much about it!